source/instructors/admin/integration/grading.rst

```diff
@@ -21,7 +29,7 @@ Example:
 
 When grades are :ref:`released <release-grades>` a URL is passed to the LMS where students/teachers can access the grading information for the assignment in the grading area of the LMS. By default, only the course teachers and the individual student can access the grading preview URL.
 
-In order to pass a different URL back to your LMS system where anyone who knows the URL can access the students grading information, enable the LMS shared feedback option for the course. To enable this, go to your course **LTI/LMS** tab and then toggle the switch to enable.
+In order to pass a different URL back to your LMS system where anyone who knows the URL can access the students grading information, enable the **LMS shared feedback** option for the course. To enable this, go to your course **LTI/LMS** tab and then toggle the switch to enable.
```

Comment:
In order to pass a different URL back to your LMS system where anyone who knows the URL -> In order to pass a different URL back to your LMS system allowing anyone who knows the URL